oxigen_crm/models/crm_lead.py

```diff
@@ -1,26 +1,32 @@
 # Copyright 2022 ForgeFlow S.L.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
 
-from odoo import models
+from odoo import api, fields, models
 
 
 class Lead(models.Model):
     _inherit = "crm.lead"
 
-    def _get_partner_email_update(self):
-        self.ensure_one()
-        generic_crm_customer = int(
+    is_generic_customer = fields.Boolean(
+        compute="_compute_is_generic_customer",
+    )
+
+    @api.depends("partner_id")
+    def _compute_is_generic_customer(self):
+        generic_crm_customer_id = int(
             self.env["ir.config_parameter"].sudo().get_param("generic_crm_customer")
         )
-        if self.partner_id.id == generic_crm_customer:
+        for rec in self:
+            rec.is_generic_customer = rec.partner_id.id == generic_crm_customer_id
```

Comment:
```suggestion
            rec.is_generic_customer = rec.partner_id.id in generic_crm_customer_ids
```
what if in the future we have 2 generic customers?

Reply:
covered in a new commit